src/components/notifications/response-notification.jsx

```diff
@@ -1,15 +1,28 @@
-import { InlineNotification } from "carbon-components-react";
+import { ToastNotification } from "carbon-components-react";
 import React from "react";
 
 export const ResponseNotification = (kind, title, message, setFunction = () =>{}) => {
+  const styles = {
+    container: {
+      position: 'fixed',
+      top: '50px',
+      right: '30px',
```

Comment:
If we use px , the element's size remains constant regardless of the size of the screen so It's better to use relative units like rem instead of px

Reply:
Preferred using px because in all other css files px is only used.

Reply:
Check the responsiveness in mobile view if it is looking fine then go ahead with px.

Reply:
<img width="1790" alt="image" src="https://github.com/Bahmni-HWC/bahmnihwc-inventory-frontend/assets/92026860/f619a4bd-d656-449a-a3d7-e6fdbc5eba02">

<img width="190" alt="image" src="https://github.com/Bahmni-HWC/bahmnihwc-inventory-frontend/assets/92026860/c597dc38-bf32-45b1-b8e1-bf6707cacef3">